### PR TITLE
Login panel with validation.

### PR DESCRIPTION
**Pull request by NatalliaKiraikova**
http://chewbacca.myjetbrains.com/youtrack/issue/App-24

### CODE REVIEW

File: test.patch [hunk 1]
```diff
@@ -0,0 +1,107 @@
+Index: frontend/app/index.html
+IDEA additional info:
```
Comment (pashapovalov) [inline test.patch]: Please remove junk file

File: frontend/app/views/header.html [hunk 2]
```diff
@@ -16,8 +16,8 @@
       </div>
       <div class="navbar-collapse collapse">
         <ul class="nav navbar-nav">
-          <li ng-class="{ active: isActive('main')}">
-            <a ng-href="#/main">My Trainings</a></li>
```
Comment (pashapovalov) [inline frontend/app/views/header.html:20]: Please, take update and check that this file doesn't contain conflict

File: frontend/app/scripts/controllers/login.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+      var rememberMeCheckbox = document.getElementById("rememberMe");
+      if (rememberMeCheckbox && rememberMeCheckbox.type == "checkbox" && rememberMeCheckbox.checked) {
+        saveToLocalStorage(this.user.email);
+      }
+      //TODO clear form after success
+    }
+  });
+
+function saveToLocalStorage(email) {
```
Comment (pashapovalov) [inline frontend/app/scripts/controllers/login.js]: please look this module https://github.com/gsklee/ngStorage

**Comment by pashapovalov** [comment]
Could you merge 5 commits to 1, please?

**Comment by kmarmaliykov** [comment]
rebased

**Pull request closed without merging**